  # Person Idenficiation Information
  # retrieved from Episodic Memory
  memory_recall:
    target_memory: 'social_events_memory'
    events: ['memory_recalled']
    data:
      human_id: []  # current tracking id
      person_id: []
      name: []
      gender: []
      eyeglasses: []
      cloth_color: []
      hair_style: []
      confidence: []
      